descqa/DeltaSigmaTest.py

```diff
@@ -0,0 +1,118 @@
+import os
+import numpy as np
+from scipy.interpolate import interp1d
+from astropy import units as u
+from astropy.coordinates import SkyCoord, search_around_sky
+import astropy.constants as cst
+from astropy.cosmology import WMAP7 # pylint: disable=no-name-in-module
+from .base import BaseValidationTest, TestResult
+from .plotting import plt
+
+__all__ = ['DeltaSigmaTest']
+
+class DeltaSigmaTest(BaseValidationTest):
+    """
+    This validation test looks at galaxy-shear correlations by comparing Delta
+    Sigma to the Singh et al (2015) measurements on the SDSS LOWZ sample.
```

Comment:
I recommend giving the link directly (http://adsabs.harvard.edu/abs/2015MNRAS.450.2195S or  https://arxiv.org/abs/1411.1755) so people don't have to search.

Reply:
good point

Reply:
Sure. I am not the expert here who knows how cosmology would impact this validation test, so it's your call.